### DISCUSSION TITLE
The Alda Docs™ as a website? (for printing, etc)

### DISCUSSION THREAD

**Discussion by halloleo** [alda-lang/alda#500] [General]
I just started to look at the Alda Docs™. :-) And I was wondering whether somebody renders these markdown files a s a normal website? Would be so much better, both to read on screen and to use for printing pages. (When I print directly from GitHub, a lot of Github "noise" gets printed with the document...)

**Comment by daveyarwood** [comment]
I've been wanting to turn the Alda docs into a website for a long time now, I've just never gotten around to doing it. If anyone reading this wants to help, please let me know!

I've made a bunch of notes about this as [issues in the alda.io repo](https://github.com/alda-lang/alda.io/issues). The last time I thought about this, I was leaning towards converting the docs into Asciidoc and using [Antora](https://antora.org/) to generate a documentation site. But it looks like recently, there are some newer options like [Docusaurus](https://docusaurus.io/) that are worth exploring. (If the tool is good enough, I don't really care strongly about whether we maintain the docs in Asciidoc vs. Markdown)

**Reply by halloleo** [reply]
Good to know.

I'm happy to have a stab at it, but it will take a while before I look into it because I have overseas holidays coming up until the end of May.
